rs/replicated_state/src/canister_snapshots.rs

```diff
@@ -156,13 +156,18 @@ impl CanisterSnapshots {
     }
 
     /// Returns the number of snapshots stored for the given canister id.
-    pub fn snapshots_count(&self, canister_id: &CanisterId) -> usize {
+    pub fn count_by_canister(&self, canister_id: &CanisterId) -> usize {
```

Comment:
Renamed to be a bit more consistent wrt the functions that exist about memory usage of snapshots as well (where we have one returning the total and one by canister id).